course_discovery/settings/base.py

```diff
@@ -635,3 +635,5 @@
     'MEDIA_ROOT': MEDIA_ROOT,
     'MEDIA_URL': MEDIA_URL
 }
+WORDPRESS_USERNAME = 'discovery_service_worker'
```

Comment:
@burhandodhy can we change the names of these settings to  something like this?
`WORDPRESS_BASIC_AUTH_USERNAME`
`WORDPRESS_BASIC_AUTH_PASSWORD`

Reply:
updated to `WORDPRESS_APP_AUTH_USERNAME` and `WORDPRESS_APP_AUTH_PASSWORD` to add the similarity with WordPress application